README.md

```diff
@@ -53,16 +244,138 @@ Optionally, use the command line tool to generate example records for Canary tes
+        
+// Create Ack from Bundle
+VRDRFhirContext ctx = new VRDRFhirContext();
+AcknowledgementMessage ackBundle = BaseMessage.parseJsonFile(AcknowledgementMessage.class, ctx, "path-to-json-data-file/AcknowledgementMessage.json");
+String bundleString = ackBundle.toJson(ctx, false);
+AcknowledgementMessage ack = BaseMessage.parseJson(AcknowledgementMessage.class, ctx, bundleString);
+```
+
+#### Create coding message
+```cs
+// Create cause of death coding message
+VRDRFhirContext ctx = new VRDRFhirContext();
+DeathRecordSubmissionMessage submission = BaseMessage.parseJsonFile(DeathRecordSubmissionMessage.class, ctx, "path-to-json-data-file/DeathRecordSubmissionMessage.json");
+CauseOfDeathCodingMessage coding = new CauseOfDeathCodingMessage(submission);
+
+// Create cause of death coding message with no identifiers
+CauseOfDeathCodingMessage coding = new CauseOfDeathCodingMessage((DeathRecordSubmissionMessage)null);
+        
+// Create cause of death coding message with no message
+DeathRecordSubmissionMessage submission = new DeathRecordSubmissionMessage();
+CauseOfDeathCodingMessage coding = new CauseOfDeathCodingMessage(submission);
+
+// Create Ack for status message
+VRDRFhirContext ctx = new VRDRFhirContext();
+StatusMessage statusMessage = BaseMessage.parseJsonFile(StatusMessage.class, ctx, "src/test/resources/json/StatusMessage.json");
+AcknowledgementMessage ack = new AcknowledgementMessage(statusMessage);
+```
+
+#### Create coding response
+```cs
+// Create cause of death coding response from json data file
+VRDRFhirContext ctx = new VRDRFhirContext();
+CauseOfDeathCodingMessage message = BaseMessage.parseJsonFile(CauseOfDeathCodingMessage.class, ctx, "path-to-json-data-file/CauseOfDeathCodingMessage.json");
+CauseOfDeathCodedContentBundle bundle = message.getCauseOfDeathCodedContentBundle();
+    // By getting list of RecordAxisCauseOfDeath from bundle
+    List<RecordAxisCauseOfDeath> recordAxis = bundle.getRecordAxisCauseOfDeath();
+
+    // By getting list of EntityAxisCauseOfDeath from bundle
+    List<EntityAxisCauseOfDeath> entityAxis = bundle.getEntityAxisCauseOfDeath();
+
+    // By getting CodingStatusValues from bundle
+    CodingStatusValues codingStatusValues = bundle.getCodingStatusValues();
+```
+
+#### Create status message
+```cs
+VRDRFhirContext ctx = new VRDRFhirContext();
+DeathRecordSubmissionMessage submission = BaseMessage.parseJsonFile(DeathRecordSubmissionMessage.class, ctx, "src/test/resources/json/DeathRecordSubmissionMessage.json");
+StatusMessage status = new StatusMessage(submission, "manualCauseOfDeathCoding");
+```
+
 # Publishing a Version
 
 To create a new release of the VRDR Java Library:
 
 1. Update the CHANGELOG.md file with information on what is changing in the release
-1. Create a pull request and merge the above changes to master
-1. Create a GitHub release
+2. Create a pull request and merge the above changes to master
```

Comment:
For Github README you can just leave as '1.' and it will automatically update the numbers, so don't change this section, if you look at the https://github.com/nightingaleproject/VRDR_javalib home you will see it's correct.